generate_wrappers.py

```diff
@@ -416,17 +428,57 @@ def generate_code(op, api_def, enum_store):
           input_names_and_types +
           attr_names_and_types +
           missing_types)]
+  body = ''
+  if len(return_name_and_types) == 0:
+    body = 'return #tfop({tfop_args})'.format(tfop_args=tfop_args)
```

Comment:
Do we really need the `return` bit in the body in this case as there are no actual output ?

Reply:
I believe so. One example is:
```swift
@inlinable @inline(__always)
public static func abort(
  errorMsg: String,
  exitWithoutError: Bool = false
) {
  return #tfop("Abort",
    error_msg: errorMsg,
    exit_without_error: exitWithoutError)
}
```

If we don't do return, type checker cannot infer the return type of `#tfop` in this context. 

Also, note the above generated swift code did not change in this PR -- the python code here is a refactoring.

Reply:
Got it, sorry for the misleading comment.